### ISSUE TITLE
Test Data Normalization

### ISSUE THREAD

**Issue by GuochaoCS** [WHU-Sigma/HyperSIGMA#9]
During the pretraining phase, all hyperspectral images (HSIs) are normalized by dividing by 4,000. In the context of testing for the super-resolution task, what standardization format should be applied to the Houston dataset? The original Houston dataset  contains 50 channels with a spatial resolution of 4172*1202.

**Comment by leonmakise** [comment]
The Houston dataset we offered has been normalized into [0,1], and we use the pre-trained model for initialization when training. 

For the testing phase, the images are also pre-processed into [0,1].
